app/models/concerns/hyku_addons/account_behavior.rb

```diff
@@ -140,5 +164,21 @@ def set_smtp_settings
           HykuAddons::PerTenantSmtpInterceptor.available_smtp_fields.each_with_object("").to_h
         )
       end
+
+      def set_shared_search_default
+        return if settings['shared_search'].present?
+        self.shared_search = false
+      end
+
+      def set_default_tenant_list
+        return if settings['tenant_list'].present?
+        self.tenant_list = []
+      end
+
+      def create_child_parent_association_from_submitted_tenant_ids
+        tenant_list.each do |tenant|
```

Comment:
Let's use Rails' scopes to help us again:
```suggestion
        self.children = Account.where(name: tenant_list).all
```

Reply:
Sorry, what does this give us?